docs/api.md

```diff
@@ -670,6 +670,24 @@ Adds a function which would be invoked in one of the following scenarios:
 
 The function is invoked after the document was created but before any of its scripts were run. This is useful to amend JavaScript environment, e.g. to seed `Math.random`.
 
+An example of overriding the navigator.languages property before the page loads:
+
+```js
+// preload.js
+module.exports = function() {
```

Comment:
if you use `readFile` method, then you shouldn't use `module.exports` part. You also don't need to define a wrapper function - the `readFile` returns a string.

So this file will simply contain the following:

```js
// preload.js  {
  get: function() {
    return ["en-US", "en", "c", "js"];
  };
});
```